CSharpSourceCode/Abilities/Crosshairs/AbilityCrosshair.cs

```diff
@@ -0,0 +1,180 @@
+﻿using System;
+using System.Collections.Generic;
+using TaleWorlds.Engine;
+using TaleWorlds.Library;
+using TaleWorlds.MountAndBlade;
+using TaleWorlds.MountAndBlade.View.Screen;
+using TOW_Core.Battle.CrosshairMissionBehavior;
+
+namespace TOW_Core.Abilities.Crosshairs
+{
+    /// <summary>
+    /// Base class for all crosshairs
+    /// </summary>
+    public abstract class AbilityCrosshair : IDisposable
```

Comment:
I believe it might be a better option to call it a  ProjectedAbilityCrosshair, to avoid confusions